.travis.yml

```diff
@@ -13,5 +13,6 @@ before_script:
 
 script:
   - docker-compose -f docker-compose.yml -f docker-compose.ci.yml run cvat_ci /bin/bash -c 'python3 manage.py test cvat/apps utils/cli'
+  - docker-compose -f docker-compose.yml -f docker-compose.ci.yml run cvat_ci /bin/bash -c 'python3 manage.py test --pattern="_tests.py" cvat/apps/dataset_manager'
```

Comment:
Why the line above doesn't work to find tests?

Reply:
Tests for the `dataset manager` app. conflict with tests for the `git` app., they are meant not to be executed in a single suite. This is done by naming tests for dm in a way that thay are not found automatically in the first line.